view/theme/frio/templates/admin/site.tpl

```diff
@@ -0,0 +1,364 @@
+
+<script>
+	$(function(){
+
+		$("#cnftheme").click(function(){
+			$.colorbox({
+				width: 800,
+				height: '90%',
+				/*onOpen: function(){
+					var theme = $("#id_theme :selected").val();
+					$("#cnftheme").attr('href',"{{$baseurl}}/admin/themes/"+theme);
+				},*/
+				href: "{{$baseurl}}/admin/themes/" + $("#id_theme :selected").val(),
+				onComplete: function(){
+					$("div#fancybox-content form").submit(function(e){
+						var url = $(this).attr('action');
+						// can't get .serialize() to work...
+						var data={};
+						$(this).find("input").each(function(){
+							data[$(this).attr('name')] = $(this).val();
+						});
+						$(this).find("select").each(function(){
+							data[$(this).attr('name')] = $(this).children(":selected").val();
+						});
+						console.log(":)", url, data);
+
+						$.post(url, data, function(data) {
+							if(timer) clearTimeout(timer);
+							NavUpdate();
+							$.colorbox.close();
+						})
+
+						return false;
+					});
+
+				}
+			});
+			return false;
+		});
+	});
+	$(function(){
+		$(".collapse").on('show.bs.collapse', function(e) {
+			localStorage.setItem('activeTab', $(e.target).attr('id'));
+		});
+		var activeTab = localStorage.getItem('activeTab');
```

Comment:
Instead of relying on `localStorage` that isn't always available, I'd suggest you pass the current `activeTab` as a hidden form input to generate the correct `/admin/site#activeTab` redirection URL that will trigger the accordion opening.

Reply:
I thought about that too, but to show a panel we need a javascript call or set the html params in the template
- A javascript call would be possible but only with regex match to get the element from the url. (Or do you know a js function to read params from the URL like `$_GET` ?)
- the template could edit the javascript code and replace the element

both ways seem more like a workaround

Reply:
We use Bootstrap forfrio, so I assume that the appropriate accordion panel will open according to the URL fragment (after the #). Please verify this before adding any custom code: http://getbootstrap.com/2.3.2/javascript.html#collapse

Reply:
I already did. That's why I started to going into js. 
`/admin/site#admin-settings-general-collapse` for the first panel group isn't working.

According to the bootstrap doc you linked it needs an controll element with
>data-toggle="collapse" and a data-target

or the manual js call `$(".collapse").collapse()`

Reply:
I found this solution:

    // Opening accordion based on URL
    var url = document.location.toString();
    if ( url.match('#') ) {
       $('#'+url.split('#')[1]).addClass('in');
    }



https://stackoverflow.com/a/24201355

What do you think? :)